res/database/Default/database_achievements.xml

```diff
@@ -27,7 +30,7 @@
 					<!-- kein Titel und es wird der Belohnungsinterne Text genommen -->
 					<title />
 					<text />
-					<data type="money" money="75000" />
+					<data type="money" money="50000" />
```

Comment:
Here, you started with an "outdated" version of the database file. Integrating your transaltion would undo changes made after the last release.